test/predicator/evaluator/variable_is_present_test.exs

```diff
@@ -0,0 +1,46 @@
+# AUTOGENERATED FILE - DO NOT EDIT
+defmodule Predicator.VariableIsPresentTest do
+  use ExUnit.Case, async: false
+  @moduletag :spec
+
+  setup_all do
+    %{instructions: [["load", "foo"], ["present"]]}
+  end
+
+  test "with_no_context", context do
+    predicate_context = nil
+    expected_result = false
+
+    result = Predicator.Evaluator.execute context[:instructions], predicate_context
+    assert expected_result == result
+    #assert_empty e.stack
+  end
+
+  test "with_blank_string", context do
+    predicate_context = %{foo: ""}
+    expected_result = false
+
+    result = Predicator.Evaluator.execute context[:instructions], predicate_context
+    assert expected_result == result
+    #assert_empty e.stack
+  end
+
+  test "with_int", context do
+    predicate_context = %{foo: 1}
+    expected_result = true
+
+    result = Predicator.Evaluator.execute context[:instructions], predicate_context
+    assert expected_result == result
+    #assert_empty e.stack
+  end
+
+  test "with_string", context do
+    predicate_context = %{foo: "bar"}
+    expected_result = true
+
+    result = Predicator.Evaluator.execute context[:instructions], predicate_context
+    assert expected_result == result
+    #assert_empty e.stack
+  end
+
```

Comment:
Using the formatter would likely be a good idea, as well as Credo. If you create a CI pipeline, `mix format --check-formatted` and `mix credo --strict` can help ensure good hygiene and good practices are maintained.